### PR TITLE
Add possibility to add custom xy-gutters if necessary

### PR DESCRIPTION
**Pull request by artursopelnik**
I will use the mixins with different gutters if necessary

### CODE REVIEW

File: scss/xy-grid/_classes.scss [hunk 1]
```diff
@@ -107,45 +107,46 @@
...
     > #{$classname} {
-      @include xy-cell-size($i, $vertical: $vertical);
+      @include xy-cell-size($i, $vertical: $vertical, $gutters: $gutters);
     }
   }
 }
 
 // Margin Grid classes
 @mixin xy-margin-grid-classes(
+  $gutters: $grid-margin-gutters,
```
Comment (DanielRuf) [inline scss/xy-grid/_classes.scss]: Better add it as last padameter.

File: scss/xy-grid/_classes.scss [hunk 2]
```diff
@@ -107,45 +107,46 @@
...
   $gutter-position: left right,
   $vertical: false,
   $wrapping-selector: '.grid-margin-x'
 ){
   #{$wrapping-selector} {
-    @include xy-gutters($negative: true, $gutter-position: $gutter-position);
+    @include xy-gutters($gutters, $negative: true, $gutter-position: $gutter-position);
```
Comment (DanielRuf) [inline scss/xy-grid/_classes.scss]: Better add it as last parameter.

**Review by DanielRuf — COMMENTED** [review]
Please also update the docs and explain which issue this resolves.

**Comment by DanielRuf** [comment]
Please use our pullrequest template. It seems you've deleted it.

**Comment by artursopelnik** [comment]
@DanielRuf where I can find the *pullrequest template*? Otherwise done.

File: docs/pages/xy-grid.md [hunk 3]
```diff
@@ -520,3 +520,25 @@ That outputs this CSS:
...
+### Multiple Grid Gutters
+
+Use the `xy-margin-grid-classes()` mixin to create classes for your own gutter.
+By default the mixin takes 4 parameters:
+- position of gutters
+- vertical direction
+- wrapping selector with your own class (.grid-margin-x-example)
+- gutters
+
+It also work with paddings by `xy-padding-grid-classes()` mixin.
```
Comment (DanielRuf) [inline docs/pages/xy-grid.md]: also work => also works
by => by using the

File: scss/xy-grid/_classes.scss [hunk 4]
```diff
@@ -127,15 +127,20 @@
   }
 }
 
-// Margin Grid classes
+/// Margin Grid classes
+///
+/// @param {List} $gutter-position [left right] - The position to apply gutters to. Accepts `top`, `bottom`, `left`, `right` in any combination.
+/// @param {Boolean} $vertical [false] - Direction of the gutters to output. See `$gutter-position`.
```
Comment (DanielRuf) [inline scss/xy-grid/_classes.scss]: Shouldn't this be a List too?

**Review by DanielRuf — CHANGES_REQUESTED** [review]

**Comment by DanielRuf** [comment]
> @DanielRuf where I can find the _pullrequest template_? Otherwise done.

See https://github.com/zurb/foundation-sites/blob/develop/.github/PULL_REQUEST_TEMPLATE.md

File: docs/pages/xy-grid.md [hunk 5]
```diff
@@ -530,7 +530,8 @@ By default the mixin takes 4 parameters:
 - wrapping selector with your own class (.grid-margin-x-example)
 - gutters
 
-It also work with paddings by `xy-padding-grid-classes()` mixin.
+It also work with paddings to by `xy-padding-grid-classes()` mixin.
```
Comment (DanielRuf) [inline docs/pages/xy-grid.md]: also work => also works
to by => sounds a bit weird, better reword this

**Review by DanielRuf — CHANGES_REQUESTED** [review]

File: scss/xy-grid/_classes.scss (hunk 4 above)
Comment (artursopelnik) [reply scss/xy-grid/_classes.scss]: I think both is correct, but boolean describe it better for $vertical

Comment (DanielRuf) [reply scss/xy-grid/_classes.scss]: boolean = true / false. This is not a direction (string).

Comment (DanielRuf) [reply scss/xy-grid/_classes.scss]: > $vertical [false] - Direction of the gutters to output. See ...

makes not much sense.

**Comment by artursopelnik** [comment]
@DanielRuf done

**Comment by DanielRuf** [comment]
We will revisit this for v7.

**Pull request closed without merging**